src-cli/witan/models/load_data.clj

```diff
@@ -196,6 +197,7 @@
   "Input should be a map with keys for each dataset and filepaths to csv
    files as the values. Output is a map of core.matrix datasets."
   [file-map]
+  (utils/property-holds? file-map map? "Not a map")
```

Comment:
Does the function still run when `file-map` is not a map? 
Should it be inside a `when` block or `try/except` block?